python/setup.py

```diff
@@ -154,10 +154,11 @@ def run(self):
                 "CMake must be installed to build the following extensions: " + ", ".join(e.name for e in self.extensions)
             )
 
-        if platform.system() == "Windows":
-            cmake_version = LooseVersion(re.search(r"version\s*([\d.]+)", out.decode()).group(1))
-            if cmake_version < "3.1.0":
-                raise RuntimeError("CMake >= 3.1.0 is required on Windows")
+        cmake_version = packaging.version.parse(
+            re.search(r"version\s*(?P<version>[\d.]+)", out.decode()).group("version")
+        )
+        if cmake_version < packaging.version.parse("3.18.0"):
+            raise RuntimeError("CMake >= 3.18.0 is required")
```

Comment:
`cmake >= 3.18` version taken from `CMakeLists.txt`.

https://github.com/openai/triton/blob/fc7c0b0e437a191e421faa61494b2ff4870850f1/CMakeLists.txt#L1

Reply:
I think the minimum version should actually be 3.20. Seems like there's a typo in the CMakeLists.txt

SET CMP0116 OLD was introduced in 3.20

Reply:
CMP0116 was introduced in 3.20, but since we are setting it to "OLD" we don't need 3.20